mappings/net/minecraft/client/gui/widget/PlayerSkinModelWidget.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/unmapped/C_rqypkmay net/minecraft/client/gui/widget/PlayerSkinModelWidget
+	FIELD f_ethsgzwv MODEL_Y_OFFSET F
+	FIELD f_ettkwxlt DEFAULT_YAW F
+	FIELD f_fdcocoaz DEFAULT_PITCH F
+	FIELD f_heswnfjk pitch F
+	FIELD f_hnhsvwhh ROTATION_SENSITIVITY F
+	FIELD f_myyhggtr renderer Lnet/minecraft/unmapped/C_rqypkmay$C_skavgwvd;
```

Comment:
Should be renamed to `skinModel` - I had originally called `PlayerSkinModel` `PlayerSkinModelRenderer`.